### ISSUE TITLE
values() is not available on EnumerableSet

### ISSUE THREAD

**Issue by olehmisar** [OpenZeppelin/openzeppelin-contracts#2825]
**💻 Environment**
Remix IDE
Openzeppelin provided by Remix IDE. Also tested with `@openzeppelin/contracts@4.2.0`.

**📝 Details**
`values()` is not available on `EnumerableSet`.

**🔢 Code to reproduce bug**
```solidity
pragma solidity 0.8.4;
import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";

contract Test {
    using EnumerableSet for EnumerableSet.AddressSet;
    EnumerableSet.AddressSet s;
    
    function f() private {
        s.length();  // works
        s.values();  // Member "values" not found or not visible after argument-dependent lookup in struct EnumerableSet.AddressSet storage ref.
    }
}

```

**Comment by olehmisar** [comment]
`EnumerableSet.values(s)` does not work either.

**Comment by Amxx** [comment]
Hello @olehmisar 

This feature is only available from version 4.3 onward (as shown in the [changelog](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/CHANGELOG.md))

You should update your version of contract if you need this feature (please not the warning that comes with it)
